deepmd/dpmodel/descriptor/dpa2.py

```diff
@@ -779,16 +900,43 @@ def serialize(self) -> dict:
                 "repformers_variable": repformers_variable,
             }
         )
+        if self.use_three_body:
+            repinit_three_body_variable = {
+                "embeddings": repinit_three_body.embeddings.serialize(),
+                "env_mat": EnvMat(
+                    repinit_three_body.rcut, repinit_three_body.rcut_smth
+                ).serialize(),
+                "@variables": {
+                    "davg": repinit_three_body["davg"],
+                    "dstd": repinit_three_body["dstd"],
+                },
+            }
+            if repinit_three_body.tebd_input_mode in ["strip"]:
+                repinit_three_body_variable.update(
+                    {
+                        "embeddings_strip": repinit_three_body.embeddings_strip.serialize()
+                    }
+                )
+            data.update(
+                {
+                    "repinit_three_body_variable": repinit_three_body_variable,
+                }
+            )
         return data
 
     @classmethod
     def deserialize(cls, data: dict) -> "DescrptDPA2":
         data = data.copy()
-        check_version_compatibility(data.pop("@version"), 2, 1)
+        check_version_compatibility(data.pop("@version"), 3, 3)
```

Comment:
I think we are still possible to support old versions just by writing some `if...else` statements.

```py
if version >= 3:
   ...
else:
   ...
```

Reply:
Do we have any breaking change here?